dockerfiles/build.sh

```diff
@@ -195,3 +195,9 @@ if [[ ${BUILD_ALL} == "true" ]]; then
 else
   build_image "${IMAGE_TO_BUILD}"
 fi
+
+if ${UPDATE_DEVFILES}; then
+  echo "> git status ----------------------------------------------------------"
```

Comment:
looks like debug instruction ?

Reply:
You can open the latest run of the action, that rebuilds the images, bumps the devfiles and creates PR https://github.com/eclipse-che/che-devfile-registry/runs/3805061526?check_suite_focus=true
Expand `Rebuild Sidecar Images` step and scroll to bottom. There is report, that a bunch of images were built.
That means, that the appropriate devfiles were bumped to sew images (build script is launched with `--update-devfiles` option)
But on the next step (Bump devfiles to new sidecars tags) git status gave nothing. It looks like the changes were disposed.

So, the idea here is to combine that both steps in one.

Reply:
I mean, can we just delete the echo output ?